### PR TITLE
Refactor Code Expression ✨

### PR DESCRIPTION
**Pull request by yezz123**
Before submitting your pull request, make sure the following is done.

* [x] Fork [the repository](https://github.com/tartiflette/tartiflette) and create your branch from `master` so that it can be merged easily.
* [ ] Update changelogs/next.md with your change (include reference to the issue & this PR).
* [x] Make sure all of the significant new logic is covered by tests.
* [x] Make sure all quality checks are green _[(Gazr specification)](https://gazr.io)_.

*[Learn more about contributing](https://github.com/tartiflette/tartiflette/blob/master/docs/CONTRIBUTING.md)*

### CODE REVIEW

File: tests/unit/test_introspection.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ async def test_introspection___schema_resolver():
     sch = await __schema_resolver(None, None, None, info)
 
     assert sch == info.schema
-    assert info.is_introspection is True
+    assert info.is_introspection
```
Comment (abusi) [inline tests/unit/test_introspection.py:17]: This is the only change I'm not really sure about, I really wanna be sure that `info.is_introspection` is a boolean and that it's value is 'True', asserting this way only assure us that it is != None or something that eval to None.

**Review by abusi — COMMENTED** [review]
Very cool changes, thank you for contributing. It'll help the project stay 'fresh' :)

File: tests/unit/test_introspection.py (hunk 1 above)
Comment (yezz123) [reply tests/unit/test_introspection.py:17]: It is unnecessary to compare boolean values to `True` or `False` in the test of an if condition. Removing these unnecessary checks makes the code slightly shorter and easier to parse.

Comment (abusi) [reply tests/unit/test_introspection.py:17]: Yeah, But this tests makes sure that `info.is_introspection` IS a boolean by asserting that it is TRUE.
Imagine tomorrow, I make a change and sudently, `info.is_introspection` is set to 32, if this is the case I want this test to fail. With your change, it'll pass.

Comment (yezz123) [reply tests/unit/test_introspection.py:17]: Yes if we are in this case then `is TRUE` is required here to bypass that issue, Looks great this idea 💡.
Let me Revert it and then we can see 🚀

Comment (yezz123) [reply tests/unit/test_introspection.py:17]: @abusi Thanks ⚡️
The Issue is fixed

Comment (yezz123) [reply tests/unit/test_introspection.py]: AA Yess i forget 😅, No problem now its fixed 🚀

**Comment by abusi** [comment]
Thanks @yezz123

**Pull request merged**